### PR TITLE
fix: echo indent

### PR DESCRIPTION
**Pull request by evilebottnawi**
Improve `echo` and `print` :star:

### CODE REVIEW

**Comment by evilebottnawi** [comment]
@czosel @mgrip ready for review. Why print and echo in one PR, because they have same logic and many places.

**Comment by evilebottnawi** [comment]
Also remove parens from arguments, all Laravel/Symfony/Yii2/PSR don't use their.

File: tests/echo/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -0,0 +1,184 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`echo.php 1`] = `
+<?php
+
+echo "Hello World";
+
+echo "This spans
+multiple lines. The newlines will be
+output as well";
```
Comment (nicoder) [inline tests/echo/__snapshots__/jsfmt.spec.js.snap]: (not the topic of this PR but) the line feeds inside this string are transformed into `\n`

this code:

```php
echo "This spans
multiple lines. The newlines will be
output as well";
```

becomes:

```php
echo "This spans\\nmultiple lines. The newlines will be\\noutput as well";
```

is this what we want?

Comment (evilebottnawi) [reply tests/echo/__snapshots__/jsfmt.spec.js.snap]: @nicoder it is bug in parser, will be fixed in new version :+1: When we update php parser we will update all snapshots :+1:

**Pull request merged**